WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift

```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
         viewModel.locationsDidChange = { [unowned self] (locations) in
             self.tableView.reloadData()
         }
-        
+        self.activityIndicatorView.isHidden = true
```

Comment:
Why this is needed when you are changing states in ViewDidAppear?

Reply:
When I ran in slow animation it is visible there unanimated when screen is loaded and disappears when the first time search is made.